_posts/2020-04-12-final-presentations-youre-all-invited.md

```diff
@@ -12,19 +12,19 @@ photos:
   license: Public domain
 ---
 
-It's been a long and exciting journey, from applications opening in October 2019, [announcing selected applicants in early January]({% link _posts/2020-01-13-ols1-announcement.md %}), and kicking off [the program's first cohort call on the 29th of January]({% link _ols-1/week02.md %}). Since then we've had an amazing range of guest speakers from around the world, sharing their expertise on topics in [the OLS syllabus]({% link ols-1.md %}), as well as a variety of breakout and discussion sessions, interspersed with mentor calls every other week. We wrapped up our learning calls with [a cohort call on self care, ally skills, and careers]({% link _ols-1/week11.md %}).  
+It's been a long and exciting journey, from applications opening in October 2019, [announcing selected applicants in early January]({% link _posts/2020-01-13-ols1-announcement.md %}), and kicking off [the program's first cohort call on the 29th of January](../openseeds/ols-1/week02.md). Since then we've had an amazing range of guest speakers from around the world, sharing their expertise on topics in [the OLS syllabus](../openseeds/ols-1.md), as well as a variety of breakout and discussion sessions, interspersed with mentor calls every other week. We wrapped up our learning calls with [a cohort call on self care, ally skills, and careers](../openseeds/ols-1/week11.md).
```

Comment:
```suggestion
It's been a long and exciting journey, from applications opening in October 2019, [announcing selected applicants in early January]({% link _posts/2020-01-13-ols1-announcement.md %}), and kicking off [the program's first cohort call on the 29th of January]({% link openseeds/ols-1/week02.md %}). Since then we've had an amazing range of guest speakers from around the world, sharing their expertise on topics in [the OLS syllabus]({% link openseeds/ols-1/index.md %}), as well as a variety of breakout and discussion sessions, interspersed with mentor calls every other week. We wrapped up our learning calls with [a cohort call on self care, ally skills, and careers]({% link openseeds/ols-1/week11.md %}).
```